db/models/app.model.js

```diff
@@ -11,5 +8,15 @@ exports.selectAllTopics = () => {
   });
 };
 
+exports.selectArticleById = (article_id) => {
+  const paramValues = [];
```

Comment:
paramValues is fine but you can just use [article_id] as the 2nd argument of db.query instead of declaring an array and pushing etc